src/utility_functions.jl

```diff
@@ -70,3 +70,84 @@ function compare(a::F, b::F, pn0 = "") where {F <: CC.Fields.Field}
+Hierarchical dictionary of available LES simulations described in [Shen2022](@cite).
+The following cfsites are available across listed models, months,
+and experiments.
+"""
+function get_LES_library()
+    LES_library = Dict("HadGEM2-A" => Dict(), "CNRM-CM5" => Dict(), "CNRM-CM6-1" => Dict())
+    Shen_et_al_sites = collect(2:15)
+    append!(Shen_et_al_sites, collect(17:23))
+    Shen_et_al_deep_convection_sites = (collect(30:33)..., collect(66:70)..., 82, 92, 94, 96, 99, 100)
+    append!(Shen_et_al_sites, Shen_et_al_deep_convection_sites)
+
+    LES_library["HadGEM2-A"]["10"] = Dict()
+    LES_library["HadGEM2-A"]["10"]["cfsite_numbers"] = setdiff(Shen_et_al_sites, [94, 100])
+    LES_library["HadGEM2-A"]["07"] = Dict()
+    LES_library["HadGEM2-A"]["07"]["cfsite_numbers"] = Shen_et_al_sites
+    LES_library["HadGEM2-A"]["04"] = Dict()
+    LES_library["HadGEM2-A"]["04"]["cfsite_numbers"] = setdiff(Shen_et_al_sites, [15, 17, 18, 32, 92, 94])
+    LES_library["HadGEM2-A"]["01"] = Dict()
+    LES_library["HadGEM2-A"]["01"]["cfsite_numbers"] = setdiff(Shen_et_al_sites, [15, 17, 18, 19, 20])
+
+    for month in ["01", "04", "07", "10"]
+        LES_library["HadGEM2-A"][month]["experiments"] = ["amip", "amip4K"]
+    end
+
+    LES_library_full = deepcopy(LES_library)
+    for month in ["01", "04", "07", "10"]
+        LES_library_full["HadGEM2-A"][month]["cfsite_numbers"] = Dict()
+        for cfsite_number in LES_library["HadGEM2-A"][month]["cfsite_numbers"]
+            if cfsite_number >= 30
+                cfsite_number = string(cfsite_number, pad = 2)
+                LES_library_full["HadGEM2-A"][month]["cfsite_numbers"][cfsite_number] = "deep"
+            else
+                cfsite_number = string(cfsite_number, pad = 2)
+                LES_library_full["HadGEM2-A"][month]["cfsite_numbers"][cfsite_number] = "shallow"
+            end
+        end
+    end
+    return LES_library_full
+end
+
+"""
+    parse_les_path(les_path)
+Given path to LES stats file on central cluster, return cfsite_number, forcing_model, month, and experiment.
+Inputs:
+ - les_path - path to les simulation containing stats folder
+Outputs:
+ - cfsite_number  :: cfsite number
+ - forcing_model :: {"HadGEM2-A", "CNRM-CM5", "CNRM-CM6-1", "IPSL-CM6A-LR"} - name of climate model used for forcing. Currently, only "HadGEM2-A" simulations are available reliably.
+ - month :: {1, 4, 7, 10} - month of simulation.
+ - experiment :: {"amip", "amip4K"} - experiment from which LES was forced.
+"""
+function parse_les_path(les_path)
+    fname = split(les_path, "/")[end]
+    fname_split = split(fname, ('.', '_'))
+    forcing_model = fname_split[3]
+    experiment = fname_split[4]
+    month = parse(Int64, fname_split[6])
+    cfsite_number = parse(Int64, replace(fname_split[2], "cfsite" => ""))
+    return (cfsite_number, forcing_model, month, experiment)
+end
+
+function valid_lespath(les_path)
+    if ~occursin(".julia/artifacts", les_path)
+        try
```

Comment:
```suggestion
        try
```
Can we please remove the try-catch?